### PR TITLE
Fixes to multi line goto and others

### PR DESCRIPTION
**Pull request by CWDN**
Couple of fixes to bugs I found while using.

1. Multi line function calls didn't work with the go to functionality.
2. If a property and a method is named the same the property would overwrite the method. Now both are suggested.
3. If your class or namespace declarations weren't all lowercase it wouldn't autocomplete.

I found that chained functions `$this->index()->search()` ..etc will go to as well, as long as the function has a `@return`.

### CODE REVIEW

File: php/services/Tools.php [hunk 1]
```diff
@@ -136,6 +136,15 @@ protected function getClassMetadata($className)
                     'descriptions' => $descriptions['descriptions']
                 )
             );
+
+            if (is_array($data['values'][$attribute->getName()])) {
+                $attributesValues = [
+                    $attributesValues,
+                    $data['values'][$attribute->getName()]
+                ];
```
Comment (Peekmo) [inline php/services/Tools.php]: Not compatible with PHP 5.3

**Comment by Peekmo** [comment]
Hello,

Thank you :+1: 

Just one thing, the PHP written should be compatible with (at least) PHP 5.3, because, even if it's no longer supported, many people are using it in old projects (I learned it from a survey). So, could you please change Tools.php and use php methods instead of just ```[ ]``` ?

As a side note, ```If your class or namespace declarations weren't all lowercase it wouldn't autocomplete.```. This issue was "volountary" because I decided to support only PHP code that follows PSR norm, and uppercase keywords is not PSR. Why this decision ? Because there is so many ways to write PHP code, to be able to parse every coding style is a hell, in term of performance and complexity. Anyway, thank you for that fix ;)

Regards,
Peekmo

**Comment by CWDN** [comment]
Hi,

Done the changes you wanted. As the project will only parse PSR standards maybe we need to show a notification to why something didn't autocomplete/jump to. This way the user isn't getting confused about why the something didn't work and then debugging and raising an issue here.

CWDN

**Comment by Peekmo** [comment]
Perhaps notification could be nice, but it need to parse the file to search for coding style issues.

Or, just recommand to use [php-cs-fixer](https://atom.io/packages/php-cs-fixer) package ? I don't know how to do it right. Anyway, it's not a problem if the package handle more than the PSR, but I don't try to handle all possible cases, if you need it, you can add it, as long as it doesn't add complexity.

Peekmo

**Pull request merged**